ui/app/components/clients/activity.ts

```diff
@@ -42,11 +45,11 @@ export default class ClientsActivityComponent extends Component<Args> {
   };
 
   get startTimeISO() {
-    return fromUnixTime(this.args.startTimestamp).toISOString();
+    return this.args.startTimestamp ? fromUnixTime(this.args.startTimestamp).toISOString() : null;
   }
 
   get endTimeISO() {
-    return fromUnixTime(this.args.endTimestamp).toISOString();
+    return this.args.endTimestamp ? fromUnixTime(this.args.endTimestamp).toISOString() : null;
   }
```

Comment:
I expected that the counts route does not provide an outlet for child routes if the activity response fails so a start and end timestamp are both required by this component. If that's not the case then the conditional in the counts page component should be adjusted since it handles the empty states.